backends-velox/src/main/scala/org/apache/spark/sql/hive/VeloxHiveUDFTransformer.scala

```diff
@@ -46,4 +40,20 @@ object VeloxHiveUDFTransformer {
       HiveUDFTransformer.genTransformerFromUDFMappings(udfName, expr, attributeSeq)
     }
   }
+
+  def isHiveUDFSupportsTransform(expr: Expression): Boolean = {
+    val (udfName, udfClassName) = getHiveUDFNameAndClassName(expr)
+    UDFResolver.UDFNames.contains(udfClassName) ||
+    UDFMappings.hiveUDFMap.contains(udfName.toLowerCase(Locale.ROOT))
```

Comment:
Is this only used for test? It seems not used in other cases.

Reply:
It's not for test. It's used here https://github.com/apache/incubator-gluten/pull/9077/files#diff-acb69e143059fbea6c76ecce0605a1343bc96c9735e6664baf1b2c439408014cR280

Reply:
oh, I mean the `UDFMappings.hiveUDFMap.contains(udfName.toLowerCase(Locale.ROOT))`. Not see it used before

Reply:
@jackylee-ch It's in `HiveUDFTransformer.genTransformerFromUDFMappings`

The `VeloxHiveUDFTransformer` will check both by udf classname first. If not, it will fallback to  `HiveUDFTransformer.genTransformerFromUDFMappings`, which will check by udf name.
<img width="919" alt="image" src="https://github.com/user-attachments/assets/2590e36b-04b7-4334-b6de-d23e8aa8f5a1" />